GitUI/CommandsDialogs/RevisionFileTreeControl.cs

```diff
@@ -655,7 +662,7 @@ private void openWithToolStripMenuItem_Click(object sender, EventArgs e)
                 var fileName = _fullPathResolver.Resolve(gitItem.FileName);
                 if (File.Exists(fileName))
                 {
-                    OsShellUtil.OpenAs(fileName.ToNativePath());
+                    OsShellUtil.OpenAs(fileName!.ToNativePath());
```

Comment:
This should not be needed with .NET5. Comment somehow?
Maybe File.Exists should be wrapped?

Reply:
I've added a comment. Perhaps all file system access should occur through an interface for testability (and diagnostics, potentially), and that interface could be correctly annotated. Probably a follow up action however.

There's also a `Microsoft.IO` package which replaces several `System.IO` operations with better performance (especially around path operations). The implementation of that interface could be implemented in terms of that, in places.